### PR TITLE
Update branch names for CI

### PR DESCRIPTION
**Pull request by elisabettai**

### CODE REVIEW

**Comment by elisabettai** [comment]
Hi @pcrespov, I  applied the changes you sent to [main.py](https://github.com/ITISFoundation/opensim-python-runner/pull/3/files#diff-7120f41bf9df684771f3a411e9513094f5be0bef68c1f2ae12f682dff2302520).

File: .github/workflows/github-ci.yml [hunk 1]
```diff
@@ -1,27 +1,25 @@
 name: build/deploy opensim-python-runner
 
 on:
+  workflow_dispatch:
```
Comment (pcrespov) [inline .github/workflows/github-ci.yml:4]: so this entry is to trigger it manually?

Comment (elisabettai) [reply .github/workflows/github-ci.yml:4]: Yes. I found it useful while testing CI. I can remove it, if you see some potential drawbacks.

Comment (pcrespov) [reply .github/workflows/github-ci.yml:4]: no, no ... i was curious. Never saw it before

**Pull request merged**